src/containers/Options/rewards.tsx

```diff
@@ -96,7 +97,7 @@ export default function Rewards({ route, navigation }) {
                 <CyDView className='mt-[10px]'>
                   <CyDText className='font-bold text-[28px]'>
                     <LoaderWithText
-                      value={rewardPoints.total.toString()}
+                      value={rewardPoints.total.toLocaleString('en-US')}
```

Comment:
don't enforce to en-US use toLocaleString() this will convert to the users local format he is in